src/test/java/Utils.java

```diff
@@ -0,0 +1,29 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.testng.Assert;
+
+public class Utils {
+    public void setPropertyDriver(){
+        System.setProperty("webdriver.chrome.driver", "src/test/resources/chromedriver.exe");
+    }
+
+    public void setSite(WebDriver driver, String site){
```

Comment:
One remark here:
Probably it's better to rename method from 'setSite' to 'openSite' to be more like 'user understandable language'